piplicenses.py

```diff
@@ -563,150 +601,190 @@ def _check_code_page(code_page):
-    parser.add_argument('--with-notice-file',
-                        action='store_true',
-                        default=False,
-                        help='when specified together with option -l, '
-                             'dump with location of license file and contents')
-    parser.add_argument('-i', '--ignore-packages',
-                        action='store', type=str,
-                        nargs='+', metavar='PKG',
-                        default=[],
-                        help='ignore package name in dumped list')
-    parser.add_argument('-o', '--order',
-                        action='store', type=str,
-                        default='name', metavar='COL',
-                        help=('order by column\n'
-                              '"name", "license", "author", "url"\n'
-                              'default: --order=name'))
-    parser.add_argument('-f', '--format',
-                        action='store', type=str,
-                        default='plain', metavar='STYLE',
-                        help=('dump as set format style\n'
-                              '"plain", "plain-vertical" "markdown", "rst", \n'
-                              '"confluence", "html", "json", \n'
-                              '"json-license-finder",  "csv"\n'
-                              'default: --format=plain'))
-    parser.add_argument('--filter-strings',
-                        action="store_true",
-                        default=False,
-                        help=('filter input according to code page'))
-    parser.add_argument('--filter-code-page',
-                        action="store", type=str,
-                        default="latin1",
-                        help=('specify code page for filtering'))
-    parser.add_argument('--summary',
-                        action='store_true',
-                        default=False,
-                        help='dump summary of each license')
-    parser.add_argument('--output-file',
-                        action='store', type=str,
-                        help='save license list to file')
-    parser.add_argument('--fail-on',
-                        action='store', type=str,
-                        default=None,
-                        help='fail (exit with code 1) on the first occurrence '
-                             'of the licenses of the semicolon-separated list')
-    parser.add_argument('--allow-only',
-                        action='store', type=str,
-                        default=None,
-                        help='fail (exit with code 1) on the first occurrence '
-                             'of the licenses not in the semicolon-separated '
-                             'list')
+        description=__summary__,
+        formatter_class=CustomHelpFormatter)
+
+    common_options = parser.add_argument_group('Common options')
+    format_options = parser.add_argument_group('Format options')
```

Comment:
The argument_group title "Format options" looks a bit strange. Because option ` --format` is in "Common options".

But I don't have a good alternative either 😅 

By the way, "Verify options" is a great naming 👍

Reply:
I agree, it's a bit strange. I can change the group names if you like, although I think it's good enough. The goal with `Format options` was to group all **additional** format arguments that one might not need that often.